test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -640,7 +640,7 @@ await CliTester.Run(new RunConfiguration
             }, _output);
         }
 
-        [Theory(Skip = "https://github.com/Azure/azure-functions-core-tools/issues/3644")]
+        [Theory]
```

Comment:
Should completion of this PR also resolve #3644 ?

Reply:
I don't think so, I believe that issue is a bit if a red herring. User secrets does work for connection strings and other settings. However, it does not work for setting "FUNCTIONS_WORKER_RUNTIME" and I believe that is a different core tools issue to the loading and utilization of user secrets.